aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -18,27 +18,36 @@ export const LayerCenterExample = () => {
             fill="vertical"
             overflow="auto"
             width={size !== 'small' ? 'medium' : undefined}
+            pad="medium"
           >
-            <Box
-              flex="grow"
-              direction="row"
-              gap="medium"
-              pad="medium"
-              height="small"
-            >
-              <Box pad={{ top: 'xsmall' }}>
-                <MailOption />
-              </Box>
-              <Box gap="medium">
-                <Heading margin="none" size="small">
+            <Box justify="between" direction="row">
+              <Box flex={false} gap="small" direction="row">
+                <Box justify="center">
+                  <MailOption />
+                </Box>
+                <Text margin="none" size="xlarge">
                   Modal Dialog
-                </Heading>
-                <Text>
-                  For modal dialogs, the use case will determine the design and
-                  size of the box for your content
                 </Text>
               </Box>
+              <Box justify="center">
+                <Button
+                  icon={<Close size="small" align="center" />}
+                  onClick={onClose}
+                />
+              </Box>
+            </Box>
+            <Box overflow="auto" pad={{ vertical: 'medium' }}>
+              <Text>
+                For modal dialogs, the use case will determine the design and
+                size of the box for your content
+              </Text>
             </Box>
+          </Box>
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={size !== 'small' ? 'medium' : undefined}
+          >
```

Comment:
Can't comment on line 54, but I think the pad should be `{{ vertical: 'small', horizontal: 'medium' }}`